bftengine/src/bftengine/messages/CheckpointMsg.cpp

```diff
@@ -68,5 +70,45 @@ void CheckpointMsg::validate(const ReplicasInfo& repInfo) const {
   // TODO(GG): consider to protect against messages that are larger than needed (here and in other messages)
 }
 
+bool CheckpointMsg::equivalent(const CheckpointMsg* a, const CheckpointMsg* b) {
+  bool equal = (a->seqNumber() == b->seqNumber()) && (a->stateDigest() == b->stateDigest()) &&
```

Comment:
it will be good to add a one-liner comment defining the equivalent Checkpoint message.

Reply:
@arc-vmware , I don't undertstand, plase clarify.